app/javascript/src/components/Dashboard/Notes/index.jsx

```diff
@@ -1,39 +1,69 @@
 import React, { useState } from "react";
 
+import { useTranslation } from "react-i18next";
+
 import Toolbar from "components/commons/ToolBar";
 
 import { SAMPLE_NOTES } from "./constants";
 import DeleteAlert from "./DeleteAlert";
 import List from "./List";
+import NewNotePane from "./Pane/Create";
+import EditNotePane from "./Pane/Edit";
 
 const Notes = () => {
+  const { t } = useTranslation();
+
   const [notes, setNotes] = useState(SAMPLE_NOTES);
   const [deleteAlertVisibliity, setDeleteAlertVisibliity] = useState(false);
-  const [selectedNoteId, setSelectedNoteId] = useState(null);
+  const [createNotePaneVisibility, setCreateNotePaneVisibility] =
+    useState(false);
+  const [editNotePaneVisibility, setEditNotePaneVisibility] = useState(false);
+  const [selectedNote, setSelectedNote] = useState(null);
 
   const handleDelete = () => {
-    setNotes(notes.filter(note => note.id !== selectedNoteId));
+    setNotes(notes.filter(note => note.id !== selectedNote.id));
+  };
+
+  const showEditNotePane = note => {
+    setEditNotePaneVisibility(true);
+    setSelectedNote(note);
   };
 
-  const showDeleteAlertForNote = id => {
+  const showDeleteAlertForNote = note => {
     setDeleteAlertVisibliity(true);
-    setSelectedNoteId(id);
+    setSelectedNote(note);
   };
 
   return (
     <div className="flex w-full flex-col">
+      <NewNotePane
+        setNotes={setNotes}
+        setShowPane={setCreateNotePaneVisibility}
+        showPane={createNotePaneVisibility}
+      />
+      <EditNotePane
+        note={selectedNote}
+        setNotes={setNotes}
+        setShowPane={setEditNotePaneVisibility}
+        showPane={editNotePaneVisibility}
+      />
       <DeleteAlert
-        handleDelete={handleDelete}
+        handleSubmit={handleDelete}
```

Comment:
This assignment does make much sense to me. Why would `handleSubmit` take value of `handleDelete`? Make sure you properly rename them.